### ISSUE TITLE
broggok lever can be reused

### ISSUE THREAD

**Issue by Aokromes** [TrinityCore/TrinityCore#28801]
### Description

at blood furnace the lever of broggok can be reused before combat with boss stars bugging the event forcing to exit from instance

### Expected behaviour

after activation lever must be disabled till wipe or kill boss.

### Steps to reproduce the problem

go blood furnace's 2nd boss
kill adds
click lever before boss moves
boss will stuck

### Branch

3.3.5

### TC rev. hash/commit

aeb4d18e20c4

### Operating system

debian 11 x64

### Custom changes

n/r

**Comment by anyanfei** [comment]
Yes, I also found the same problem,after activation lever,the monster won't come out of the door because the door is still closed

**Comment by CraftedRO** [comment]
the lever seems reated to trash mobs captive in wall not with the boss gate anyway I found that if you leave combat you can spawn those mobs over and over again

**Comment by CraftedRO** [comment]
91473e7f5cd0e09b2788dabcc7740f5480aed484 I'm unable to reproduce the main ticket issue with a normal level account have no idea if https://github.com/TrinityCore/TrinityCore/commit/ad3fc81db9cfb899fc72dfef175cd897c9a4b05c did a difference here or not

**Comment by Aokromes** [comment]
yeah can't reproduce on head.
